scoped-transitions.md

```diff
@@ -1,130 +1,167 @@
+The steps for a scoped view transition are based on the
+[steps for a document view transition](https://drafts.csswg.org/css-view-transitions-1/#lifecycle)
+with appropriate modifications.  At a high level:
 
-   This allows the developer to update scoped-transition-root's subtree without presenting it to the user. See [Suppressing rendering](#suppressing-rendering) for details.
+1. Create the [`ViewTransition`](https://drafts.csswg.org/css-view-transitions-1/#viewtransition) object.
 
-3. The captured output above is painted in a stacking context which is a descendant of scoped-transition-root's stacking context. The above example generates the following stacking context hierarchy.
+2. At the next rendering opportunity, capture the painted output of each tagged
+   element under the scoped transition root, and create the pseudo-element tree
+   with `::view-transition-old` pseudo-elements.  A tagged element's geometry
+   information is computed relative to the scoped transition root.
 
-   ```
-   root
-   └─ outer
-      ├─ inner
-      └─ outer-page-transition (pseudo-tree underneath)
-   ```
+3. Invoke the callback passed to `startViewTransition`.
 
-   If the scoped-transition-root is tagged, it's snapshot uses [content-capture](https://github.com/WICG/shared-element-transitions/blob/main/explainer.md#more-granular-style-capture) mode. This means that only the element's descendants (including both text and elements) are painted in the snapshot while box decorations (like border, box-shadow) continue to paint in the outer stacking context. See [root snapshot](#root-snapshot) for details.
+4. Create the `::view-transition-new` pseudo-elements and set up the default
+   animations.
 
-4. Once the callback passed to `startViewTransition` is finished, incoming-image pseudo-elements are added to the tree generated in step 2. Default animations are added as defined in [7.8](https://drafts.csswg.org/css-shared-element-transitions-1/#animate-a-page-transition-algorithm).
-
-## Constraints
+5. Run the animations.
 
-An element which is the root for a transition has the following constraints. These constraints may stop being satisfied at any phase of the transition lifecycle and we'll need to define the fallback behavior for such cases. The simplest option is to consider them a developer error and [skip the transition](https://drafts.csswg.org/css-shared-element-transitions-1/#skip-the-page-transition).
+6. Clean up by destorying the pseudo-element tree.
 
-1. There shouldn't be an active transition on the element, its descendants or its ancestors. This is because a transition involves taking the painted output of an element and rendering it into a pseudo-element. For example, in the following case:
+Between steps 2 and 4, we need to pause the rendering of the scoped transition
+root's subtree so that the developer can update it without presenting those
```

Comment:
I think you mean the UA?

```suggestion
root's subtree so that the user-agent can update it without presenting those
```

Reply:
Actually I meant the developer since the updates occur in the DOM callback.

Reply:
Well yeah, it’s the UA doing the update by invoking the callback that the author specified.

“…  so that the developer can update it …” reads to my as the developer manually doing some DOM manipulations in between those steps.

Maybe rephrase to the following then:

```suggestion
root's subtree so that the user-agent can execute the callback without presenting those
```

?

Reply:
Tweaked the wording, PTAL

Reply:
Perfect!